src/usyms.h

```diff
@@ -37,7 +37,30 @@ class Usyms {
   std::map<std::string, std::map<uintptr_t, elf_symbol, std::greater<>>>
       symbol_table_cache_;
 
+  void cache_bcc(const std::string& elf_file);
+  std::string resolve_bcc(uint64_t addr,
+                          int32_t pid,
+                          const std::string& pid_exe,
+                          bool show_offset,
+                          bool show_module);
   struct bcc_symbol_option& get_symbol_opts();
+
+#ifdef HAVE_BLAZESYM
+  struct blaze_symbolizer* symbolizer_{ nullptr };
+
+  struct blaze_symbolizer* create_symbolizer() const;
+  void cache_blazesym(const std::string& elf_file);
+  std::optional<std::string> resolve_blazesym_int(uint64_t addr,
```

Comment:
I know the name is the same in ksyms but maybe this is slightly better `resolve_blazesym_inst` ? Non blocking

Reply:
No idea what "inst" signifies but I suppose you will say the same thing about "int". So sure, why not.

Reply:
Hah - I thought "int" was short for "instruction" was it not?

Reply:
I typically use it to mean "internal". As in, an implementation detail not visible outside the module.

Reply:
Ah, the convention I'm used to for such things is "impl" e.g. `resolve_blazesym_impl` but NBD.

Reply:
Okay, will rename then.

Reply:
@danielocfb Sorry to be a pain but can we at least make it consistent with ksyms? I'm fine changing it back to "int" or "impl" or whatever.

Reply:
Yep, done.